packages/mobile/src/screens/personal-info/components/profile-image/profile-image.tsx

```diff
@@ -1,31 +1,104 @@
-import React, { FC } from 'react';
+/* eslint-disable no-extra-boolean-cast */
+import React, { FC, useState, useCallback } from 'react';
+import { Image } from 'react-native';
+import { image as imageService, notification } from '~/services/services';
 import {
   View,
   UserIcon,
   Pressable,
   CameraIcon,
   FlagBackgroundView,
+  PhotoPickerModal,
+  Spinner,
 } from '~/components/components';
+import { requestCameraStoragePermission } from '~/permissions/android-permissions';
 import { globalStyles } from '~/styles/styles';
+import { pickImageCamera, pickImageLibrary } from '~/helpers/helpers';
 import { styles } from './styles';
 
 const ProfileImage: FC = () => {
+  const [showModal, setShowModal] = useState(false);
+  const [photoUri, setPhotoUri] = useState('');
+  const [isPhoto, setIsPhoto] = useState(false);
```

Comment:
I assume you may use only one oh these, because the following seems to be correct` isPhoto === Boolean(photoUri)`

Reply:
updated